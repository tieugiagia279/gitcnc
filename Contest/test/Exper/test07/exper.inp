100
21966 8391
25177 6950
6405 2010
22042 6349
74629 201
83149 6453
46256 3730
69473 6613
7163 2153
64749 9162
72315 9609
83860 2986
33072 2852
93046 2999
99915 6982
15369 2253
76505 6856
19870 5890
26038 4634
17842 8657
15819 6447
14688 205
50994 3635
90774 4895
33381 6540
9716 5315
6344 1311
77565 1243
71624 2705
1286 8271
89091 8331
20011 2797
95347 2325
7467 2620
2098 8919
69046 4847
60274 8866
81929 5334
777 4198
51728 6374
87861 9286
84058 1867
46014 8656
94492 1667
46343 4558
5330 9345
63347 9729
2631 74
38047 3327
51924 7868
8401 522
96052 3871
92941 8682
83588 9946
38565 5648
1249 4255
68440 8060
8757 8061
90921 191
99567 5473
55461 6277
35925 344
66422 1796
31674 7057
90730 1674
81254 4390
20326 5884
58320 4207
75332 5401
32445 9184
58876 3994
77315 6278
99521 3814
68861 5171
42223 5247
96784 3405
73769 4006
37882 2426
93614 931
64647 1249
66076 2456
87534 6975
14813 2224
54373 3868
45521 6810
55346 5219
7790 1027
6962 4711
32068 780
13477 6499
2645 3934
29195 5742
52623 2174
70078 3777
91787 9317
48120 4685
3331 4047
26905 2443
70366 7544
21797 395
