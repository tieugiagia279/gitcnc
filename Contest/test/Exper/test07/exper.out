9
7
3
3
2
5
2
6
3
7
13
4
2
7
6
3
8
4
5
9
6
2
4
4
9
3
2
4
2
3
4
2
4
5
5
3
7
7
2
5
6
4
13
5
5
13
11
2
5
11
4
9
10
14
7
2
9
10
2
7
7
5
3
6
2
6
5
3
4
20
3
6
4
3
5
5
4
4
4
2
3
4
4
4
5
4
3
6
2
8
4
6
4
2
8
5
3
3
7
2
